assets/terraform-modules/aws/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -173,3 +173,22 @@ variable "enable_csi" {
   type        = bool
   default     = false
 }
+
+
+variable "cpu_manager_policy" {
+  description = "CPU Manager policy to use for the worker pool. Possible values: `none`, `static`."
+  default     = "none"
+  type        = string
+}
+
+variable "kube_reserved_cpu" {
+  description = "CPU cores reserved for the Worker Kubernetes components like kubelet, etc."
+  default     = "300m"
+  type        = string
+}
+
+variable "system_reserved_cpu" {
+  description = "CPU cores reserved for the host services like Docker, sshd, kernel, etc."
+  default     = "1500m"
```

Comment:
I still think this is too high.

Reply:
Let us start somewhere and adjust value as we see more data around this?

Reply:
Made it 500m.